src/main/java/com/kodilla/ecommercee/domain/Order.java

```diff
@@ -1,41 +1,65 @@
 package com.kodilla.ecommercee.domain;
 
+import lombok.*;
+
 import javax.persistence.*;
 import javax.validation.constraints.NotNull;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
 
 /**
  * Author Kamil Seweryn
  */
-@Entity
-@Table(name = "ORDERS")
+@Entity(name = "orders")
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
 public class Order {
     @Id
-    @GeneratedValue(strategy = GenerationType.AUTO)
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     @NotNull
     @Column(name = "id", unique = true)
     private Long id;
-    private String name;
 
-    public Order() {
-    }
+    @NotNull
+    @Column(name = "status")
+    @Enumerated(EnumType.STRING)
+    private Status status;
 
-    public Order(String name) {
-        this.name = name;
-    }
+    @NotNull
+    @Column(name = "order_date")
+    private LocalDate orderDate;
 
-    public Long getId() {
-        return id;
-    }
+    @Column(name = "send_date")
+    private LocalDate sendDate;
 
-    public String getName() {
-        return name;
-    }
+    @Column(name = "delivered_date")
+    private LocalDate deliveredDate;
+
+    @OneToMany(targetEntity = Product.class,
```

Comment:
Na pewno ma tak zostać? W tym momencie nie zrobisz czegoś w stylu:

Klient A robi zamówienie na Produkt 1
Klient B robi zamówienie na Produkt 1

Klient B nie dostanie zamówienia bo produkt 1 jest przypisany do klienta A
A jak ktoś będzie chciał zamówić 1000x produkt 1 to będziesz musiał zrobić 1000 takich samych produktów co trochę nie ma sensu...

Reply:
no tak, tutaj powinno być ManyToMany albo nowa encje OneToMany OrderRow i tam ManyToOne na produkt

Reply:
Poprawione. :-) Czy takie rozwiązanie jest ok?